.github/workflows/ci.yml

```diff
@@ -23,17 +23,15 @@ jobs:
         run: forge build
 
       - name: Run tests
-        run: forge test -vvv --fork-url ${{ secrets.MAINNET_RPC }}
+        run: forge test -vvv
```

Comment:
Think you want to keep this file the same.

Reply:
Could explain your reasoning?

Reply:
You're right. Disregard. For some reason, I was thinking this was in the vault-v2 repo when I initially reviewed.